packages/driver/src/cypress.ts

```diff
@@ -551,7 +551,7 @@ class $Cypress {
 
         // this event is how the reporter knows how to display
         // stats and runnable properties such as errors
-        this.emit('test:after:run', ...args)
+        this.emit('test:after:run', args[0], this.config('isInteractive'))
```

Comment:
Was there intentions to release these logs in the [runner.js logic ](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/runner.ts#L1738) as well?

We are storing these log values twice.

Reply:
We don't retain the logs in run mode in the runner: https://github.com/cypress-io/cypress/blob/53db0ea71f756cec93efd8cd0c95218aac9b893a/packages/driver/src/cypress/runner.ts#L1761